src/components/view/GarminView/GarminView.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useState } from 'react'
-import { Layout, Card, StatusBarBackground, ConnectGarmin, GarminDevices, GarminMonthCharts } from "../.."
+import { Layout, Card, StatusBarBackground, ConnectGarmin, GarminDevices, GarminMonthCharts, LoadingIndicator } from "../.."
```

Comment:
I don't see a LoadingIndicator in this view?

Reply:
Its down on Line 41.